frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html

```diff
@@ -1,12 +1,31 @@
 <div class="container">
+    <button class="create-lesson-button" (click)="openCreate()">+ Add Speaking Club</button>
+
     <div class="date-container">
-        <h2 class="date">05 March 2023, Tuesday</h2>
+        <h2 class="date">{{ todayDate }}</h2>
     </div>
 
     <div class="lessons-section" *ngIf="lessons">
         <div class="lesson" *ngFor="let lesson of lessons">
             <app-lesson [lesson]="lesson"></app-lesson>
         </div>
     </div>
-    <button (click)="openCreate()"></button>
+
+    <div class="no-lessons" *ngIf="lessons.length === 0">
+        <p *ngIf="selectedTopicsFilters.size === 0 && selectedLanguageFilters.size === 0">
+            Oops, there are no lessons for this day. <br> Please consider another date
+        </p>
+
+        <p *ngIf="selectedTopicsFilters.size !== 0 && selectedLanguageFilters.size !== 0">
+            Oops, there are no lessons with such filters for this day. <br> Please consider another date
+        </p>
+
+        <p *ngIf="selectedTopicsFilters.size !== 0 && selectedLanguageFilters.size === 0">
+            Oops, there are no lessons with such topics for this day. <br> Please consider another date
+        </p>
+
+        <p *ngIf="selectedTopicsFilters.size === 0 && selectedLanguageFilters.size !== 0">
+            Oops, there are no lessons with such levels for this day. <br> Please consider another date
+        </p>
```

Comment:
It will be better to do through function. Just return a text
And 0 is false, >0 - true, so no need to add === / !==